cli/azd/pkg/project/importer.go

```diff
@@ -20,13 +19,11 @@ import (
 
 type ImportManager struct {
 	dotNetImporter *DotNetImporter
-	console        input.Console
 }
 
-func NewImportManager(dotNetImporter *DotNetImporter, console input.Console) *ImportManager {
```

Comment:
It's added by me in previous PR. Now I found it's not necessary, so I deleted it.